deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -469,6 +469,9 @@ rules:
   - apiGroups: [""]
     resources: ["serviceaccounts"]
     verbs: ["get"]
+  - apiGroups: ["csiaddons.openshift.io"]
```

Comment:
this should be a clusterRole or do you need only Roles?

Reply:
as you are creating the csiaddons object in the namespace where the rook operator is running it should be a role. Have you tried with roles instead of cluster roles?

Reply:
no didn't try with roles yet. Will give it a try

Reply:
@Madhu-1 Adding roles works fine for me. I will be updating the PR with roles and also adding testing results in the comment for reference.